platform/kube/client.go

```diff
@@ -406,3 +406,29 @@ func (cl *Client) GetTLSSecret(uri string) (*model.TLSSecret, error) {
 		PrivateKey:  key,
 	}, nil
 }
+
+// Request sends requests through the Kubernetes apiserver proxy to
+// the a Kubernetes service.
+// (see https://kubernetes.io/docs/concepts/cluster-administration/access-cluster/#discovering-builtin-services)
+func (cl *Client) Request(namespace, service, method, path string, inBody []byte) ([]byte, error) {
+	absPath := fmt.Sprintf("api/v1/namespaces/%s/services/%s/proxy/%s/%s",
+		namespace, service, IstioResourceVersion, path)
+	var status int
+	outBody, err := cl.dyn.Verb(method).
```

Comment:
`dyn`/`rest.RESTClient` has the HTTP verb methods. `cl.client`/`kubernetes.Clientset` doesn't. I could have created an additional RESTClient with Group/Version for k8s.io/v1 just for this purpose, but setting the absolute path seemed to work just as well. We should probably refactor the k8s platform client anyway and parameterize Group/Version since we don't need TPR access in istioctl anymore with proper API server. Then we could replace AbsPath() here with Suffix().